packages/xstate-react/src/createXStateContext.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { createContext, useContext } from 'react';
+import {
+  InterpreterFrom,
+  MachineOptions,
+  StateFrom,
+  StateMachine
+} from 'xstate';
+import { useActor } from './useActor';
+import { useInterpret } from './useInterpret';
+import { useSelector } from './useSelector';
+
+export function createXStateContext<
+  TMachine extends StateMachine<any, any, any, any>
+>(machine: TMachine) {
+  const context = createContext({} as InterpreterFrom<TMachine>);
+
+  function createSelector<T>(selector: (state: StateFrom<TMachine>) => T) {
+    return selector;
+  }
+
+  const useXStateContext = (): InterpreterFrom<TMachine> => {
```

Comment:
q: when this would be useful publicly if we have `useActor` and `useSelector` helpers here? for what kind of a task they are not enough?

Reply:
For when you want to send a running service an event, but not subscribe to updates to it or select any data from it.